add backup-integration in Core (#26530)
home-assistant/home-assistant.github.io: source/_docs/configuration.markdown

## Migrating to a new system

> The preferred way of migrating to a new system is by backup using {% my supervisor_backups title="supervisor backup" %}. Then install new system (HAOS or Supervisor), and restore backup at first logon.

Reviewer: I don't think changing the link to "supervisor backup" is correct, it might be confused with taking a backup of just the supervisor (which is of course not the case).

```suggestion
The preferred way of migrating to a new system is by backup using {% my supervisor_backups title="backup" %}. Then install the new system (HAOS or Supervisor), and restore the backup during the onboarding steps.
```

> If you run the container or core installation method you don't have supervisor-backup and will need to use the {% my backup title="backup-integration" %} to make a backup.

Reviewer: I now see the problem you are trying to solve with the "supervisor backup". Maybe instead, we should write something like:

> The preferred way of migrating to a new system is by making a backup. Backups can be found and created in the Settings menu, next go to **System** -> **Backup**.

The my links might be more confusing, so maybe just leaving them out, in this case, is an option?
